# **Mito-COI**
##  **Mito-COI database**

This is a non-redundant database based on NCBI that contains all eukaryotic mitochondria and COI genes. It contains almost all mitochondrial sequences and COI sequences as of _**December 2021**_.

# The database components provided by this GitHub Repository
  See pdf for the composition of the Venn diagram: venn.pdf (The number indicates the sequence number)
## Mito-COI  
  A non-redundant reference database of all mitochondrial and COI sequences for eukaryotes.
## Mitochondrial  
  A non-redundant reference database containing only mitochondrial-related sequences for eukaryotes.
## COI  
  A non-redundant reference database containing only COI-related sequences for eukaryotes.
## 3xia.genus  
  A characteristic database created with 57 common zooplankton genus recorded in the Three Gorges Reservoir area from historical records (not used in the article), for **tutorials** and **testing**.
  Smaller files are suitable for uploading to Github.

# Abbreviations used in this GitHub Repository and their meanings
* mito/mitochondrial  
  Mainly focus on keywords "Mitochondrion" and "Mitochondrial"
* COI  
  Mainly focus on keywords "COI" and "COX1"
* 3xia.genus  
  More than 50 common genus in the Three Gorges Reservoir area
* Euka/Eu  
  eukaryotic/eukaryotes
* acc  
  NCBI accession ID, GenBank number
* tax  
 taxonomy information
* taxid  
  The classification number (ID) corresponding to each accession ID on NCBI
* .qza  
  qza format  files used for qiime2 analysis, including classification information files and reference sequence files
* ref  
  reference
* refseq  
 reference sequences in the Database
* rmdup  
  Remove duplicate sequences based on 100% sequence similarity
* lab  
  Title corresponding to each sequence on NCBI
* .fasta  
  fasta format file

# Tutorial on how to use and build a custom database
Please visit Folder **"TutorialOnHowToUseAndBuildACustomDatabase"** to learn how to use these databases we created, and create your own unique database.

# Citation
1）Hu, H., Wei, X.-Y., Liu, L., Wang, Y.-B., Jia, H.-J., Bu, L.-K. and Pei, D.-S. (2023) Supervised machine learning improves general applicability of eDNA metabarcoding for reservoir health monitoring. _**Water Research**_ 246, 120686.      
**doi**:10.1016/j.watres.2023.120686
            
2）Hu, H., Wei, X.-Y., Liu, L., Wang, Y.-B., Bu, L.-K., Jia, H.-J. and Pei, D.-S. (2024) Biogeographic patterns of meio- and micro-eukaryotic communities in dam-induced river-reservoir systems. _**Applied Microbiology and Biotechnology**_ 108(1), 1-17.      
**doi**:10.1007/s00253-023-12993-4      
# Email
huhuan@cigit.ac.cn


